macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionDebugMenu.swift

```diff
@@ -103,6 +103,9 @@ final class NetworkProtectionDebugMenu: NSMenu {
 
                 NSMenuItem(title: "Reset Site Issue Alert", action: #selector(NetworkProtectionDebugMenu.resetSiteIssuesAlert(_:)))
                     .targetting(self)
+
+                NSMenuItem(title: "Reset VPN Disable Exclusion Suggestions", action: #selector(NetworkProtectionDebugMenu.resetVPNDisableExclusionSuggesitons(_:)))
+                    .targetting(self)
```

Comment:
Helper for testing my changes.  This allows the user to reset the "Don't ask again" toggle.